part-configuration.tex

```diff
@@ -60,6 +60,21 @@ \subsection{ledgersmb.yaml}
 \chapter{Per company configuration}
 \label{cha-company-config}
 
+\section{Matching your business processes}
+\label{sec-company-config-matching-your-business}
+
+Currently, LedgerSMB operates such that all optional functionality is available and the user decides to use it or not, provided they have proper application roles (see \appref{app-role-listing}).

```

Comment:
Maybe add a remark that not-assigning roles is an effective way of hiding functionality from users if the application is to be simplified?